interface/new/new_comprehensive.php

```diff
@@ -948,6 +948,14 @@ function end_group()
     <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
     <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
   });
+  $('.datepicker-past').datetimepicker({
+    <?php $datetimepicker_timepicker = false; ?>
+    <?php $datetimepicker_showseconds = false; ?>
+    <?php $datetimepicker_formatInput = true; ?>
+    <?php $datetimepicker_maxDate = '+1970/01/01'; ?>
+    <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
+    <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
+  });
```

Comment:
hi @im-Amitto ,
This is really nice work overall, but I think this means that each form will need to contain a datepicker-past (and future and for the combination if need to support both with time and without time) which I think results in 4 more classes or I think the datepicker will not work. I'm not sure how to solve this unless @mdsupport has an idea to do that?

Reply:
Upon thinking about this, the layouts date picker is only used in a finite number of places (maybe 6 or so), so if did need to hard-code the 4 classes it definitely wouldn't be the end of the world to need to add these:
datepicker-past
datetimepicker-past
datepicker-future
datetimepicker-future

Reply:
by hardcode do you mean that i should add these four classes to all files using them or i am interpreting it wrong.

Reply:
yep, I think that is the only way to support that feature and for them to not break. There will then be 6 classes. After finding all the places you need to put these, I bet we can centralize them into a call anywhere that uses the layouts templating.

Reply:
datetimepicker is currently present in 130 files.

Reply:
I might be totally wrong here but why are we not making certain variables optional in here? We could technically code it with object-oriented programming techniques...